README.md

```diff
@@ -1 +1,75 @@
-# WeatherApp
\ No newline at end of file
+# WeatherApp
+
+# WeatherApp (JavaScript)
+
+This project is a weather app that allow user to search for weather by the name of the city.
+
+## Live Demo
+
+[Link here](https://wonderful-pare-04b09a.netlify.app/)
+
+## ScreenShoot
+
+![screenshoot](dist/assets/images/screenshot1.png)
+![screenshoot](dist/assets/images/screenshot2.png)
+
+## Built With
+
+- JavaScript
+- ESLint
+- VS Code
+
+## Prerequisites
+
+You must have a browser on your PC in order to use the application. [Click here](https://www.mozilla.org/en-US/firefox/new/) to download one.
+
+## Getting Started
+
+1. clone the project
+
+```bash
+# Clone this repository
+$ git clone https://github.com/rida-elbahtouri/WeatherApp.git
+
+# Go into the repository
+$ cd WeatherApp
+
+# start the app
+Copy the absolute path of the index file, paste on your browser and click on enter
+```
+
+## for webpack run:
+
+```bash
+$ npm run build
+
+$ npm run start
```

Comment:
- [ ] Your project has a package.json file and dependencies listed in it. Make sure to include the needed npm instalations.